src/components/App/AppIFrame.js

```diff
@@ -43,6 +43,7 @@ class AppIFrame extends React.Component {
   static propTypes = {
     app: AppType.isRequired,
     iframeRef: PropTypes.func,
+    instancePath: PropTypes.string,
```

Comment:
Is this just for bookkeeping, so we can inspect it later? It doesn't seem super useful to include here (and the state doesn't get updated if the prop changes).

Reply:
Indeed! Removed https://github.com/aragon/aragon/pull/846/commits/cb8fcf2352fbdf016da9deb167246876da93ab70

Reply:
It doesn't look like the prop's used at all; should we just remove it entirely?

Reply:
Oh right!